projects/elife.yaml

```diff
@@ -745,6 +746,17 @@ generic-cdn:
             default-ttl: 86400 # seconds
             headers:
                 - Origin
+        fastly:
+            subdomains:
+                - "{instance}-fastly"
+            backends:
+                # first is default
+                default:
+                    hostname: elife-cdn.s3.amazonaws.com
+                articles:
+                    hostname: "{instance}-elife-published.s3.amazonaws.com"
+                    condition: req.url ~ "^/articles" 
```

Comment:
Should this have a trailing slash?

Reply:
Can be added, works but can be made more specific. It's pure VCL